span-normalizer/span-normalizer/src/main/java/org/hypertrace/core/spannormalizer/jaeger/JaegerSpanNormalizer.java

```diff
@@ -63,6 +64,11 @@
   private final JaegerResourceNormalizer resourceNormalizer = new JaegerResourceNormalizer();
   private final TenantIdHandler tenantIdHandler;
 
+  // measure the span's start time and its processing time
+  private static final String DELAY_IN_SPAN_PROCESSED_TIME_METRIC = "span.processed.delay.time";
```

Comment:
```suggestion
  private static final String SPAN_PROCESSING_DELAY_TIME_METRIC = "span.processing.delay.time";
```

Reply:
Done via UI to handle in a single commit.